lib/Wikia/tests/Service/User/Auth/HeliosCookieHelperTest.php

```diff
@@ -24,6 +24,8 @@ public function setUp() {
 			->disableOriginalConstructor()
 			->getMock();
 
+		$this->request = $this->getMock( 'WebRequest', [ 'getCookie', 'getHeader' ] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getCookie" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getHeader" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)